scripts/referrals.test.ts

```diff
@@ -111,11 +111,11 @@ describe('fetchReferralEvents', () => {
     ])
     expect(mockGetRegistryContract).toHaveBeenCalledTimes(2)
     expect(mockGetRegistryContract).toHaveBeenCalledWith(
-      '0x0',
+      '0x5a1a1027aC1d828E7415AF7d797FBA2B0cDD5575',
```

Comment:
kinda weird to use actual production addresses here.  feels unsafe.  is there an easy way to avoid it?

Reply:
Removed the contract address in af01c0a2eb8b761c9588f99d9bc0cad8989f2178.